opdr1.js

```diff
@@ -0,0 +1,44 @@
+//A
+
+const typeNamesTv = inventory.map((tv) => {
+    return tv.type;
+})
+
+console.log(typeNamesTv);
+
+
+//B
+
+const tvIsSoldOut = inventory.filter((tv) => {
+  return tv.originalStock === tv.sold;
+})
+
+console.log(tvIsSoldOut);
+
+
+//C
+
+const tvWithAmbilight = inventory.filter((tv) => {
+    return tv.options.ambiLight === true;
+})
```

Comment:
Deze zou je nog iets kunnen versimpelen door `=== true` weg te laten
```suggestion
const tvWithAmbilight = inventory.filter(tv => tv.options.ambiLight);
```